LanguageFeatures/Augmentation-libraries/defining_augmentation_A04_t01.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion It is a compile-time error if:
+/// ...
+/// - An augmentation library contains any part directives.
+///
+/// @description Checks that it is a compile-time error if an augmentation
+/// library contains any part directives.
```

Comment:
OK, this might change (Lasse wants to enable it, and I'd support that), but it is clearly specified to be an error at this time, so we'll go with that and then change the test if the rules are changed.